resources/public/api/conf/1.0/checkpoints_list.yaml

```diff
@@ -1,7 +1,7 @@
 summary: List checkpoints
 description: |
-  This endpoint allows a developer to get all the checkpoints created for the vendor ID. 
-  A National Insurance number can be provided to filter the results.
+  This endpoint enables you to get all the checkpoints created for the vendor ID. 
+  A National Insurance number can be provided to to filter the results based on National Insurance number.
```

Comment:
Duplicate `to` here and in the tech spec also. I assume this is not meant to be here!

Reply:
Good spot!